kubernetes/ocfweb-worker.yaml.erb

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ocfweb-worker-deployment
+  labels:
+    app: ocfweb-worker
+spec:
+  replicas: 1
```

Comment:
:thinking:

Reply:
(🤔)[https://github.com/ocf/services/blob/090b1ef6c09d82b0f1713d47ffe7692a2fb50356/apps/ocfweb/worker/app.yaml#L8]

Reply:
also, i think the reasoning is since it just writes stuff to redis, even if the worker goes down the site is still live. also having multiple workers doing the same computations and persisting them to redis (one potentially overwriting the other) is not particularly helpful.